examples/cloudrun/mysql/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Use the official Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.25 as builder
+
+# Create and change to the app directory.
+WORKDIR /app
+
+# Retrieve application dependencies.
+# This allows the container build to reuse cached dependencies.
+# Expecting to copy go.mod and if present go.sum.
+COPY go.* ./
+RUN go mod download
+
+# Copy local code to the container image.
+COPY . .
+
+# Build the binary.
+# CGO_ENABLED=0 builds a statically-linked binary.
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o server
+
+# Use the official Alpine image for a lean production container.
+# https://hub.docker.com/_/alpine
+# https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+FROM alpine:latest
```

Comment:
Go samples should use the distroless base image: gcr.io/distroless/static:nonroot